### DISCUSSION TITLE
Teams

### DISCUSSION THREAD

**Discussion by apprenticius** [Terrance/SkPy#216] [Q&A] [answered]
If I send a message with my Skype account to a Microsoft Teams user over the WEB GUI all works fine, but if I do the same this code here then I get an error. Sendung messages to Skype recipients all woks well. Is there any restriction?
Can I receive webrtc SDP with this code to forward it to another webrtc client?

**Comment by Terrance** [comment, marked as answer]
> I send a message with my Skype account to a Microsoft Teams user

This isn't a use case I've tested, so it likely needs some changes to make it work.

> I get an error

This would be best tracked via a [GitHub issue](https://github.com/Terrance/SkPy/issues/new) -- that page will direct you on how to properly describe the problem and the errors you may be seeing.

> Can I receive webrtc SDP

I don't know what SDP is, but SkPy handles messaging only (i.e. text and attachments) -- voice/video calling is not included.

**Reply by apprenticius** [reply]
My Intension was to build a bridge (xep-0115 component) to my XMPP-Server. Messages are feasible, but I also would like to use the voip functionality.

Ok, anyway thanks a lot for you friendly answer.
